src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -202,12 +204,13 @@ public function handle()
             "frame_code"                 => $this->getResource("views/frame_code.html.php"),
             "env_details"                => $this->getResource("views/env_details.html.php"),
 
+            "preface"        => $this->templateHelper->escape($plainTextException),
             "title"          => $this->getPageTitle(),
             "name"           => explode("\\", $inspector->getExceptionName()),
             "message"        => $inspector->getExceptionMessage(),
             "docref_url"     => $inspector->getExceptionDocrefUrl(),
             "code"           => $code,
-            "plain_exception" => Formatter::formatExceptionPlain($inspector),
```

Comment:
Not sure we can delete it, but deprecate at least for removal on ntext major version